\hypertarget{classnumpy_1_1lib_1_1__iotools_1_1ConverterLockError}{}\doxysection{numpy.\+lib.\+\_\+iotools.\+Converter\+Lock\+Error Class Reference}
\label{classnumpy_1_1lib_1_1__iotools_1_1ConverterLockError}\index{numpy.lib.\_iotools.ConverterLockError@{numpy.lib.\_iotools.ConverterLockError}}


Inheritance diagram for numpy.\+lib.\+\_\+iotools.\+Converter\+Lock\+Error\+:\nopagebreak
\begin{figure}[H]
\begin{center}
\leavevmode
\includegraphics[width=236pt]{classnumpy_1_1lib_1_1__iotools_1_1ConverterLockError__inherit__graph}
\end{center}
\end{figure}


Collaboration diagram for numpy.\+lib.\+\_\+iotools.\+Converter\+Lock\+Error\+:\nopagebreak
\begin{figure}[H]
\begin{center}
\leavevmode
\includegraphics[width=236pt]{classnumpy_1_1lib_1_1__iotools_1_1ConverterLockError__coll__graph}
\end{center}
\end{figure}


\doxysubsection{Detailed Description}
\begin{DoxyVerb}Exception raised when an attempt is made to upgrade a locked converter.\end{DoxyVerb}
 

The documentation for this class was generated from the following file\+:\begin{DoxyCompactItemize}
\item 
inventaria\+\_\+env/lib/python3.\+10/site-\/packages/numpy/lib/\+\_\+iotools.\+py\end{DoxyCompactItemize}
